code/game/objects/items/handcuffs.dm

```diff
@@ -33,9 +33,15 @@
 	materials = list(MAT_METAL=500)
 	breakouttime = 600 //Deciseconds = 60s = 1 minute
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 50)
+	can_suicide = TRUE
 	var/cuffsound = 'sound/weapons/handcuffs.ogg'
 	var/trashtype = null //for disposable cuffs
 
+//pretty sure swalloing these would choke you
+/obj/item/restraints/handcuffs/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is swalloing [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")

```

Comment:
Swallowing

Reply:
how about swallowing the tongue or holding the breath?